src/main/resources/application.yaml

```diff
@@ -41,27 +41,33 @@ spring:
             token-uri: https://hmctsdartsb2csbox.b2clogin.com/hmctsdartsb2csbox.onmicrosoft.com/B2C_1_darts_externaluser_signin/oauth2/v2.0/token
             jwk-set-uri: https://hmctsdartsb2csbox.b2clogin.com/hmctsdartsb2csbox.onmicrosoft.com/B2C_1_darts_externaluser_signin/discovery/v2.0/keys
 
-#  datasource:
```

Comment:
`application-local.yaml` has the following comment:
```
# Once the postgres config is uncommented in the main application.yml the below config may not be needed
```
So it may be worth reviewing if `application-local.yaml` needs to be updated in line with these changes.